### PR TITLE
[Liftoff Monetize] 7.2.0.0 - Update error objects to use the Liftoff errors where applicable

### PR DESCRIPTION
**Pull request by stanley-vungle**
The Liftoff error codes has been added to public documentation.

Please update Google's public documentation for Liftoff SDK to refer to this page for the error codes:
https://support.vungle.com/hc/en-us/articles/15661036969627

Thanks.

### CODE REVIEW

File: adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMediationAdapterVungle.h [hunk 1]
```diff
@@ -18,17 +18,8 @@
 typedef NS_ENUM(NSInteger, GADMAdapterVungleErrorCode) {
```
Comment (stanley-vungle) [inline adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMediationAdapterVungle.h:18]: The other error codes are handled within the SDK. The two remaining ones issues when retrieving valid information or object from the adapter from the Google SDK

File: adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleInterstitial.m [hunk 2]
```diff
@@ -75,12 +74,6 @@ @implementation GADMediationVungleInterstitial {
 }
 
 - (void)requestInterstitialAd {
-  if (!self.desiredPlacement.length) {
```
Comment (hikaruwatanabe-eng) [inline adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleInterstitial.m:78]: Is there any reason we want to remove it? Looks like it still can be  @""

Comment (stanley-vungle) [reply adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleInterstitial.m:78]: It's ok if it is an empty string. The SDK has been updated to be able to handle these cases and will properly return a failed to load callback when the load is requested.

Comment (hikaruwatanabe-eng) [reply adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleInterstitial.m:78]: Ah okay! Thank you for the explanation.

File: adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleRewardedAd.m [hunk 3]
```diff
@@ -92,6 +85,13 @@ - (void)loadRewardedAd {
   VungleAdsExtras *extras = [[VungleAdsExtras alloc] init];
   [extras setWithWatermark:[_adConfiguration.watermark base64EncodedStringWithOptions:0]];
   [_rewardedAd setWithExtras:extras];
+    if ([_adConfiguration extras] &&
+        [[_adConfiguration extras] isKindOfClass:[VungleAdNetworkExtras class]]) {
+      VungleAdNetworkExtras *extras = (VungleAdNetworkExtras *)[_adConfiguration extras];
```
Comment (hikaruwatanabe-eng) [inline adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleRewardedAd.m]: I think this should be named networkExtras or should be refactored into a separate function/method like `findPlacement:`. One can get confused with extras declared at line 92/85.

Comment (stanley-vungle) [reply adapters/LiftoffMonetize/LiftoffMonetizeAdapter/Bidding/GADMediationVungleRewardedAd.m]: Ok. Updated

**Pull request merged**